apps/back/src/Twig/AppExtension.php

```diff
@@ -10,13 +10,16 @@ class AppExtension extends AbstractExtension
     public function getFilters(): array
     {
         return array(
-            new TwigFilter('divide_by_100', array($this, 'divideBy100')),
+            new TwigFilter('format_currency', array($this, 'formatCurrency')),
         );
     }
 
-    public function divideBy100($amount): float
+    public function formatCurrency($number, $decimals = 2, $decPoint = '.', $thousandsSep = ','): string
     {
-        $amount = $amount/100;
+        $number = $number / 100;
```

Comment:
```suggestion
        $number = $number / 100;
```
Du coup tu divise toujours, il faut directement les caster en string et mettre ta virgule sans faire de division.

L'idée, si tu as une dépense à 1,45€ les étapes sont :
```php
$expense = 145;    // ton entrée
$expense = "145";  // la valeur initiale convertie en string
$expense = "1,45€"; // la virgule ajoutée en laissant deux caractères à la fin
```

Tu peux t'aider de la fonction [substr_replace](https://www.php.net/substr_replace) 👍

Reply:
je n'avais pas pensé à quelque chose comme ça :notbad: